includes/import.form.inc

```diff
@@ -1240,3 +1286,185 @@ function islandora_multi_importer_startprocessing($parameters) {
   // islandora-multi-importer-zeroMQboot > /dev/null &");
   // islandora_multi_importer_client_task();
 }
+
+/**
+ * Implements hook_form_alter.
+ *
+ * This function implements hook_form_alter with intent to set defaults
+ * for islandora_multi_importer_form fields.
+ *
+ * @param $form
+ * @param $form_state
+ * @param $form_id
+ */
+function islandora_multi_importer_form_alter(&$form, &$form_state, $form_id) {
```

Comment:
can this be actually set directly on the form builder function? ideas on maybe keeping presets saved? via a select box?